apps/platform/src/pages/DiseasePage/DiseaseAssociations/DiseaseAssociations.jsx

```diff
@@ -14,34 +14,10 @@ import {
   DataUploader,
 } from "../../../components/AssociationsToolkit";
```

Comment:
Perfect 👍 can you export the AotfApiPlayground component from the AssociationsToolkit index? It can be imported as part of the AotfToolkit instead of a fixed route